pennylane/ops/op_math/controlled_ops.py

```diff
@@ -791,8 +789,7 @@ def _primitive_bind_call(cls, wires, id=None):
         return cls._primitive.bind(*wires, n_wires=2)
 
     def __init__(self, wires, id=None):
-        control_wire, wire = wires
-        super().__init__(qml.PauliX(wires=wire), control_wire, id=id)
+        super().__init__(qml.PauliX(wires=wires[1:]), wires[:1], id=id)
```

Comment:
Could make this consistent with Y, Z
```suggestion
        super().__init__(qml.X(wires=wires[1:]), wires[:1], id=id)
```

Reply:
don't really want to restart all the CI...